OsmAnd/src/net/osmand/data/FavouritePoint.java

```diff
@@ -54,12 +103,16 @@ public PointDescription getPointDescription() {
 	}
 
 	public boolean isPersonal() {
-		return false;
+		return PERSONAL_CATEGORY.equals(category);
 	}
 
 	@Override
-	public PointDescription getPointDescription(Context ctx) {
-		return getPointDescription();
+	public PointDescription getPointDescription(@NonNull Context ctx) {
+		if (isPersonal()) {
```

Comment:
not clear difference between personal and non-persona. I think it should be identical in both cases return new PointDescription(PointDescription.POINT_TYPE_FAVORITE, getName(ctx));